src/commands/org/api.ts

```diff
@@ -92,13 +101,14 @@ export class OrgApi extends SfCommand<void> {
     const headers: { [key: string]: string } = {};
 
     for (const header of keyValPair) {
-      const split = header.split(':');
-      if (split.length !== 2) {
+      const [key, ...rest] = header.split(':')
+      const value = rest.join(':').trim();
```

Comment:
I was copying headers from chrome dev tools and notice that `"Origin:https://gus.lightning.force.com"` would not parse correctly because of the additional `:`